chapter_multilayer-perceptrons/environment_vn.md

```diff
@@ -451,7 +463,9 @@ The confusion matrix C, is simply a $k \times k$ matrix where each column corres
 Each cell's value $c_{ij}$ is the fraction of predictions where the true label was $j$ *and* our model predicted $y$.
 -->
 
-*dịch đoạn phía trên*
+Để ước lượng phân phối của nhãn mục tiêu, đầu tiên ta dùng một bộ phân loại tương đối tốt (thường được học trên tập huấn luyện) và sử dụng một tập kiểm định (cùng phân phối với tập huấn luyện) để tính ma trận confusion.
+Ma trận confusion C là một ma trận $k \times k$, trong đó mỗi cột tương ứng với một nhãn *thật* và mỗi dòng tương ứng với nhãn dự đoán của mô hình.
+Còn giá trị của mỗi phần tử $c_{ij}$ là tỉ lệ dự đoán với nhãn thật là $j$ *và* nhãn dự đoán là $y$.
```

Comment:
tác giả lại nhầm rồi, lại còn tự nhiên in nghiêng từ `and` nữa chứ :)))
```suggestion
Giá trị của mỗi phần tử $c_{ij}$ là tỉ lệ mẫu có nhãn thật là $j$ *và* nhãn dự đoán là $i$.
```

Reply:
Đoạn này mình thấy để từ `Còn `ở đầu để nối tiếp câu bên trên vì để không thì khá rời rạc. Có thể đây là ý kiến cá nhân thôi.

Trong bản gốc của sách mình thấy có in nghiêng từ `and`, có thể hàm ý nhấn mạnh quan hệ `và` chăng?

`Each cell's value $c_{ij}$ is the fraction of predictions where the true label was $j$ *and* our model predicted $y$.`